SecretAdmin/Features/Program/ProgramIntroduction.cs

```diff
@@ -31,6 +31,11 @@ public static void ShowIntroduction()
 
             // Server Options
             
+            Log.Alert("Do you want to auto-install EXILED? (y/n)");
+            var ai = System.Console.ReadLine()?.ToLower();
+            if (ai != null && (ai[0] == 'y' || ai[0] == 'n'))
```

Comment:
ai[0] == 'n', and installs Exiled?
Am I not seeing something

Reply:
I may have pulled a smoothbrain 😔 
Jesus fixed this in a later commit